doc/release_notes/release_notes_2.5.rst

```diff
@@ -169,17 +169,14 @@ formatting, and presentation throughout the ACRN documentation:
 Fixed Issues Details
 ********************
 
-.. comment example item
-   - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
-
 - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
 - :acrn-issue:`5879` - hybrid_rt scenario does not work with large initrd in pre-launched VM
-- :acrn-issue:`6015` - HV and DM: Obsolete terms cleanup for SSRAM
+- :acrn-issue:`6015` - HV&DM： Obsolete terms cleanup for SSRAM
 - :acrn-issue:`6024` - config-tools: generate board_info.h and pci_dev.c using xslt
 - :acrn-issue:`6034` - dm: add allow_trigger_s5 mode to pm_notify_channel uart
 - :acrn-issue:`6038` - [REG][RAMDISK] Fail to launch pre RTVM while config ramdisk
 - :acrn-issue:`6056` - dm: a minor bug fix of unregister_mem_int
-- :acrn-issue:`6072` - [WHL][WAAG]use config tool to passthru Audio,will not display GOP
+- :acrn-issue:`6072` - [WHL][WAAG]use config tool to passthru Auido,will not display GOP
```

Comment:
I had made some spelling fixes that your PR "unfixed".  Can you limit this PR to just the known issues section.